S2VX.Game/Editor/Containers/EditorInfoBar.cs

```diff
@@ -42,12 +42,6 @@ public class EditorInfoBar : CompositeDrawable {
             TextAnchor = Anchor.TopCentre,
         };
 
-        public VolumeDisplay VolumeDisplay { get; } = new() {
-            Anchor = Anchor.TopRight,
-            Origin = Anchor.TopRight,
-            TextAnchor = Anchor.TopRight,
-        };
```

Comment:
Need to also remove menu items related to volume

Also, adjusting volume with alt + mouse scroll in the command panel doesn't seem to be working for me

Reply:
Ok menu items removed.

Lol the scroll container was only counting as the scroll bar so volume scrolling only worked there. I added onscroll to the whole command panel so it should be working now.

Reply:
Good catch, looks good now